specs/v1.yaml

```diff
@@ -821,6 +1059,13 @@ components:
                   The final grant amount in cents that will be processed by Chariot and submitted to the DAF.
                   This amount must be in whole dollar increments (rounded to the nearest hundred) as currently
                   all DAFs only accept whole dollar grants.
+              applicationFeeAmount:
+                type: number
+                description: |-
+                  This parameter specifies the fee your platform plans to take from the grant in cents. This
+                  parameter only applies for grants made through the DAFpay Network. Before
+                  the payment is made available to the nonprofit, Chariot's fee and the applicationFeeAmount
+                  is deducted. The sum of Chariot's fee and the applicationFeeAmount cannot exceed 5% of the grant's amount.
```

Comment:
is this validation done upfront by our server before submitting the grant? if so, what does this API fail with in the case that this happens, e.g. 400 status code? it's useful to clearly write out the error cases